lib/cpl.rb

```diff
@@ -186,6 +189,18 @@ def self.all_base_commands
     rescue StandardError => e
       ::Shell.abort("Unable to load command: #{e.message}")
     end
+
+    private
+
+    def puts_info_header(config)
+      rows = {}
+      rows["ORG"] = config.org unless config.org.nil? || config.org.empty?
+      rows["APP"] = config.app unless config.app.nil? || config.app.empty?
```

Comment:
Do we need to check for `nil` and `empty` or would `if config.org` and `if config.app` work?

Reply:
What if we get an empty string for `config.app`? It is a truthy value.
I think in such cases, it is important to have a strict condition.
I am even thinking of stripping the value. Something like this:

```ruby
rows["ORG"] = config.org.to_s.strip unless config.org.to_s.strip.empty?
```

Reply:
Also, I am wondering what is the right behavior if no org or app is provided. Currently, I don't show them. But shouldn't we put some message like `no <KEY> found!`?

Reply:
If we want to strip it, it might be best to do that when assigning the app/org in the `Config` class, and then we can make sure that it's always going to be either a string or `nil`. That way we can standardize how they are accessed across the entire code.

And I don't know about adding a "not found" message, because in some commands, the org/app is optional, so it might confuse users to see "not found", leading them to believe they should have specified it. Maybe it would be best to use "not provided"?

```
ORG: not provided
APP: not provided
```

Reply:
If the user doesn't provide either of `org` or `app`, we get error messages before reaching to the point of showing the info header:

```
➜  CPLN_ORG="   my-org  " cpl deploy-image 
No value provided for required options '--app'
➜  CPLN_APP=tutorial-app cpl deploy-image 
ERROR: Can't find option 'cpln_org' for app 'tutorial-app' in 'controlplane.yml', and CPLN_ORG env var is not set. The org can also be provided through --org.
```

But the issue is with the following scenario:
```
➜ CPLN_ORG="    " CPLN_APP=tutorial-app cpl deploy-image
ORG: 
APP: tutorial-app
```

It is beyond the scope of this PR to fix this issue.
We should have a proper mechanism for all configurations.

What do you think?